test/integration/fn_mount_cmd_test.go

```diff
@@ -116,7 +116,7 @@ func validateMountCmd(ctx context.Context, t *testing.T, profile string) { // no
 	}
 
 	start := time.Now()
-	if err := retry.Expo(checkMount, time.Millisecond*500, Seconds(15)); err != nil {
+	if err := retry.Expo(checkMount, 5*time.Second, Seconds(15)); err != nil {
```

Comment:
I think the 5*Second is the amount of time that waits first time before retrying, again, and 15 is total time. is that what you wanted?

Reply:
ah thanks! accidentally left that in from debugging